src/components/AnonymousChat/child/ChatItemTargetText.tsx

```diff
@@ -63,6 +74,60 @@ const styles = StyleSheet.create({
   },
   mr8: {
     marginRight: 4
+  },
+  attachmentContainer: {
+    width: '100%',
+    height: 268,
+    borderRadius: 8,
+    overflow: 'hidden',
+    flexDirection: 'row',
+    flexWrap: 'wrap'
+  },
+  image: {
+    width: '100%',
+    height: '100%',
+    resizeMode: 'cover'
+  },
+  moreOverlay: {
+    position: 'absolute',
+    backgroundColor: 'rgba(0,0,0,0.5)',
+    width: '100%',
+    height: '100%',
+    justifyContent: 'center',
+    alignItems: 'center'
+  },
+  moreText: {
+    fontSize: dimen.normalizeDimen(16),
+    fontFamily: fonts.inter[400],
+    color: COLORS.white
+  },
+  attachmentFileContainer: {
+    backgroundColor: COLORS.gray,
+    justifyContent: 'center',
+    height: dimen.normalizeDimen(64),
+    flexDirection: 'row',
+    alignItems: 'center'
+  },
+  attachmentFileContent: {
+    flex: 1,
+    padding: dimen.normalizeDimen(6)
+  },
+  attachmentFileName: {
+    fontSize: dimen.normalizeDimen(14),
+    fontFamily: fonts.inter[600],
+    color: COLORS.black
+  },
+  attachmentFileInfo: {
+    fontSize: dimen.normalizeDimen(12),
+    fontFamily: fonts.inter[400],
+    color: COLORS.gray500
+  },
+  attachmentFileIcon: {
+    backgroundColor: COLORS.light_silver,
+    width: dimen.normalizeDimen(64),
+    height: '100%',
+    alignItems: 'center',
+    justifyContent: 'center'
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

You've added a lot of new styles for handling attachments. It's good to see that you're using constants for dimensions, which will help maintain consistency across different screens and devices. However, it would be better if you could move these styles into a separate file to keep this component file cleaner and more focused on its main purpose.



<!-- This is an auto-generated comment by OSS CodeRabbit -->